eng/targets/Helix.Common.props

```diff
@@ -16,7 +16,7 @@
     <HelixAvailableTargetQueue Include="Windows.7.Amd64.Open" Platform="Windows" />
     <HelixAvailableTargetQueue Include="Windows.10.Amd64.EnterpriseRS3.ASPNET.Open" Platform="Windows" EnableByDefault="false" />
     <HelixAvailableTargetQueue Include="OSX.1012.Amd64.Open" Platform="OSX" />
-    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="OSX" />
+    <HelixAvailableTargetQueue Include="OSX.1014.Amd64.Open" Platform="OSX" EnableByDefault="false" />
```

Comment:
Can you see what happens when you remove this line entirely? I think that's the reason you are getting an error on helix.

Reply:
This list is used to list all the helix queues that are used as well as which helix queues are used to test per each project. If we want to completely remove this platform that would work, but if we want to enable this queue for some subset of tests, then we need to keep this here and mark it with EnableByDefault="false". This is why `Windows.10.Amd64.EnterpriseRS3.ASPNET.Open` looks the same.

I'm making the changes with the assumption that we actually have a set of tests to run on OSX 10.14. If there are no such tests then yes, we should remove the queue completely.

Reply:
I'm advocating for what unblocks helix and removes the red X entirely. My concern is that having even just one agent enabled will cause the test run not to complete as, presumably, other teams are still using these mac agents too.

>I'm making the changes with the assumption that we actually have a set of tests to run on OSX 10.14. If there are no such tests then yes, we should remove the queue completely.

Eventually yes, we should run all tests on OSX 10.14, but until we have more agents, we should remove the workload entirely.